tests/integration/test_writes.py

```diff
@@ -672,69 +643,3 @@ def test_table_properties_raise_for_none_value(
             session_catalog, identifier, {"format-version": format_version, **property_with_none}, [arrow_table_with_null]
         )
     assert "None type is not a supported value in properties: property_name" in str(exc_info.value)
-
-
-@pytest.mark.integration
-@pytest.mark.parametrize("format_version", [1, 2])
```

Comment:
Moved the metadata tables to their own tests file to avoid blowing up this one :)